converter/src/test/java/gov/cms/qpp/acceptance/XpathJsonPathComparisonTest.java

```diff
@@ -13,6 +12,7 @@
 import java.nio.file.Paths;
 
 public class XpathJsonPathComparisonTest {
+
```

Comment:
Was this intended to not be converted to use junit 5?

Reply:
Done